File: legacy-runtime/src/main/java/com/xiaomi/smack/packet/StreamError.java
package com.xiaomi.smack.packet;

import com.xiaomi.push.mpcd.Constants;

/* JADX INFO: loaded from: miuipushsdkshared_3_7_9.jar:com/xiaomi/smack/packet/StreamError.class */
public class StreamError {
    private String code;

    public StreamError(String str) {
        this.code = str;
    }

    public String getCode() {
        return this.code;
    }

    public String toString() {
        return "stream:error (" + this.code + Constants.SEPARATOR_RIGHT_PARENTESIS;
    }
}

File: legacy-runtime/src/main/java/com/xiaomi/smack/packet/StreamError.kt
package com.xiaomi.smack.packet

/*
 * Current override reference: com.xiaomi.xmsf 0.3.17-20260410000745 (versionCode 1003003000),
 * base.apk sha256 f3d72b6f5e1427ceecd3147a051d58e4dc95bb528397d486658e01cad9f7e590,
 * JADX path: com.xiaomi.xmsf/current/base/sources/com/xiaomi/smack/packet/StreamError.java
 * No stock 7.4.67-C same-path source was found in the split source tree.
 */
class StreamError(val code: String?) {

    override fun toString(): String {
        return "stream:error ($code)"
    }
}
